### PR TITLE
added pure frontend launch pages

### PR DESCRIPTION
**Pull request by danielbrog**
no back end integration or preview

### CODE REVIEW

**Comment by danielbrog** [comment]
This is my first time using material ui, if there's any feedback in terms of best practices or things to do better I would appreciate it.

File: client/src/components/FileUpload.js [hunk 1]
```diff
@@ -9,12 +9,13 @@ const baseStyle = {
   padding: "20px",
   borderWidth: 2,
   borderRadius: 2,
-  borderColor: "#eeeeee",
+  borderColor: "#69E781",
   borderStyle: "dashed",
```
Comment (aeciorc) [inline client/src/components/FileUpload.js:14]: Use the theme primary color instead

File: client/src/pages/launch.js [hunk 2]
```diff
@@ -1,7 +1,249 @@
...
+  },
+  textField: {
+    marginBottom: theme.spacing(3),
+    minWidth: 415
+  },
+  emphasis: {
+    border: "none",
+    width: "80px",
+    borderRadius: "5px",
+    borderTop: "5px solid #69E781"
```
Comment (aeciorc) [inline client/src/pages/launch.js]: Same as above

File: client/src/components/FileUpload.js (hunk 1 above)
Comment (aeciorc) [reply client/src/components/FileUpload.js:14]: Add a "cursor: pointer" property

File: client/src/pages/launchdetails.js [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+          </Typography>
+
+          {/* Preview Button */}
+          <Button
+            className={classes.button}
+            type="button"
+            variant="contained"
+            type="button"
+            onClick={() => {
+              alert("Preview showing...");
```
Comment (aeciorc) [inline client/src/pages/launchdetails.js]: Get rid of this alert and the one below

**Review by aeciorc — APPROVED** [review]
You did a good job, actually. Just a couple pointers below:

File: client/src/pages/launch.js (hunk 2 above)
Comment (aeciorc) [reply client/src/pages/launch.js]: (for the color)

**Review by asadsid95 — APPROVED** [review]

**Pull request merged**